.changeset/gentle-tips-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/compiler': minor
+---
+
+changing raw_with_expression_loop in tokenizer to only handle string that has '`' differently otherwise it should treat it as normal string
```

Comment:
I don't see the code for this changeset on this branch. Remove?

Reply:
yep, good catch